src/schema/mood.js

```diff
@@ -0,0 +1,43 @@
+import { resolveManifestation, resolveWork } from "../utils/utils";
+
+export const typeDef = `
+  enum MoodFieldType {
+      TITLE
+      CREATOR
+      MOODTAGS
+      ALL
+   }
+   
+   type MoodSearchResponseType {
+    """   
+    Title(s) of work - is set when debug is true
+    """
+    title: [String!]
+    
+    """
+    Creator(s) of work - is set when debug is true
+    """
+    creator: [String!]
+    
+    """
+    The work
+    """
+    work: Work!
+   }
+   
+   type MoodSearchResponse {
+
+    """
+    Response is an array of MoodSearchResponseType
+    """
+    response: [MoodSearchResponseType!]!
```

Comment:
Synes vi skal lave svaret så det passer så godt som muligt med vores andre søgedimser. Kan vi tilføje hitcount, for at understøtte paginering